### DISCUSSION TITLE
"Fix the following" block of info

### DISCUSSION THREAD

**Discussion by EvgenyKisel** [TroyWalshProf/SeleniumAxeDotnet#142] [Q&A] [answered]
Hey @TroyWalshProf and all contributors! Deque added a new block to results details at chrome extension sometime ago (or it was from the very first version, and I just didn't notice it). 
![image](https://user-images.githubusercontent.com/21366158/114865718-53032800-9dfb-11eb-896d-2140d0689cd3.png)
It has more usefull information for dev team to fix violation, but it is not attached to their help links. To my mind it will be helpful to add such info into report that you generates for violations. What do you think? Is it smth that you can do soon?

**Comment by TroyWalshProf** [comment, marked as answer]
Like it will add soon.

**Reply by EvgenyKisel** [reply]
Thanks!
